app/src/main/java/io/apicurio/registry/storage/impl/sql/SQLServerSqlStatements.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.storage.impl.sql;
+
+/**
+ * H2 implementation of the sql statements interface.  Provides sql statements that
+ * are specific to PostgreSQL, where applicable.
+ * @author eric.wittmann@gmail.com
+ */
+public class SQLServerSqlStatements extends CommonSqlStatements {
```

Comment:
For @Apicurio/developers Would it make sense to create a common class that PostgreSQL and SQL Server statements would inherit from? This differs from PostgreSQL in 3 added methods. It may be too much inheritance though:)

Reply:
As you prefer :)

Reply:
We'll think about this and clean it up post-merge if we think it makes sense.  @matthyx don't worry about it. :)